test/append.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var eq = require('./utils').eq;
+var S = require('..');
+
+
+describe('append', function() {
+
+  it('adds the element to the end of the list', function() {
+    eq(S.append('c', ['a', 'b']), ['a', 'b', 'c']);
+    eq(S.append({x: 3}, [{x: 1}, {x: 2}]), [{x: 1}, {x: 2}, {x: 3}]);
+  });
+
+  it('adds a list to a list of lists', function() {
+    eq(S.append([3, 4], [[1], [2]]), [[1], [2], [3, 4]]);
+  });
+
+  it('works on empty list', function() {
+    eq(S.append(1, []), [1]);
+  });
+
+});
```

Comment:
It would be nice to assert that the arguments may be provided one at a time:

```javascript
it('is curried', function() {
  eq(S.append('c').length, 1);
  eq(S.append('c')(['a', 'b']), ['a', 'b', 'c']);
});
```